lib/NotificationListener.js

```diff
@@ -123,6 +123,11 @@ function NotificationListener(localEndpoint) {
         });
     });
 
+    sax.on('tag:groupmute', (property) => {
+      logger.trace(property.$text);
+      _this.emit('group-mute', uuid, property.$text);
+    });
+
     sax.on('tag:containerupdateids', (property) => {
```

Comment:
Have you verified that this is emitted as a groupmute tag, and not as part of a last change tag? Because Volume is evented as part of a LastChange event, and embedded within that tag.